api/groups/eventsData.go

```diff
@@ -23,28 +50,82 @@ func UnmarshallBlockDataV1(marshalledData []byte) (*data.SaveBlockData, error) {
 }
 
 // UnmarshallBlockData will try to unmarshal block data
-func UnmarshallBlockData(marshaller marshal.Marshalizer, marshalledData []byte) (*data.ArgsSaveBlockData, error) {
-	var argsBlockS *outport.OutportBlock
-	err := json.Unmarshal(marshalledData, &argsBlockS)
+func (edh *eventsDataHandler) UnmarshallBlockData(marshalledData []byte) (*data.ArgsSaveBlockData, error) {
+	var outportBlock *outport.OutportBlock
+	err := json.Unmarshal(marshalledData, &outportBlock)
```

Comment:
```suggestion
	err := edh.marshaller.Unmarshal(marshalledData, &outportBlock)
```
?

with this suggestion, code coverage can be improved as well with custom data returned here

Reply:
added separate marshaller for internal marshalling